main/src/main/scala/sbt/internal/PluginsDebug.scala

```diff
@@ -121,17 +121,33 @@ private[sbt] object PluginsDebug {
     if (Project.isProjectLoaded(s)) {
       val extracted = Project.extract(s)
       import extracted._
-      def helpBuild(uri: URI, build: LoadedBuildUnit): String = {
-        val pluginStrings = for (plugin <- availableAutoPlugins(build)) yield {
-          val activatedIn =
-            build.projects.toList.filter(_.autoPlugins.contains(plugin)).map(_.id)
-          val actString =
-            if (activatedIn.nonEmpty) activatedIn.mkString(": enabled in ", ", ", "")
-            else "" // TODO: deal with large builds
-          s"\n\t${plugin.label}$actString"
-        }
-        s"In $uri${pluginStrings.mkString}"
-      }
+
+      def helpBuild(uri: URI, build: LoadedBuildUnit): String =
```

Comment:
The build `uri` parameter is not used here. Should we start output with a line like
```
In build ${uri}:
  Enabled plugins in module1 (/Users/aabramov/dev/open-source/sbt-3212/module-1):
    sbt.plugins.CorePlugin
    sbt.plugins.Giter8TemplatePlugin
    ...
  Enabled plugins in module2 (/Users/aabramov/dev/open-source/sbt-3212/module-2):
    sbt.plugins.CorePlugin
    sbt.plugins.Giter8TemplatePlugin
    ...
  Enabled plugins in module3 (/Users/aabramov/dev/open-source/sbt-3212/module-3):
    sbt.plugins.CorePlugin
    sbt.plugins.Giter8TemplatePlugin
    ...
```
?

Reply:
Maybe it's better to print the full absolute path once at the top as build URI, and then skip the `(/Users/aabramov/dev/open-source/sbt-3212/module-1)`?

Reply:
Agree.
So let it be
```
In build /Users/aabramov/dev/open-source/sbt-3212:
  Enabled plugins in module1:
    sbt.plugins.CorePlugin
    sbt.plugins.Giter8TemplatePlugin
    ...
  Enabled plugins in module2:
    sbt.plugins.CorePlugin
    sbt.plugins.Giter8TemplatePlugin
    ...
  Enabled plugins in module3:
    sbt.plugins.CorePlugin
    sbt.plugins.Giter8TemplatePlugin
    ...
```
, am I right?

Reply:
Something like this:
![image](https://user-images.githubusercontent.com/11317222/95085727-499c5300-0728-11eb-8d29-08be3df03dbc.png)